cloud-tenant-base-dependencies-enforcer/pom.xml

```diff
@@ -94,6 +95,8 @@
                                         <include>com.sun.xml.bind:jaxb-core:[${jaxb.version}]:jar:provided</include>
                                         <include>com.sun.xml.bind:jaxb-impl:[${jaxb.version}]:jar:provided</include>
                                         <include>commons-logging:commons-logging:[1.1.3]:jar:provided</include>
+                                        <include>jakarta.activation:jakarta.activation-api:[1.2.1]:jar:provided</include>
+                                        <include>jakarta.xml.bind:jakarta.xml.bind-api:[2.3.2]:jar:provided</include>
```

Comment:
Intentional ? I do not see any use of this package.

Reply:
These are new transitive dependencies from `vespa_jersey2` (from some updated jersey-jackson integration artifact).

Reply:
K